BannerlordTwitch/BLTAdoptAHero/BLTAdoptAHero.cs

```diff
@@ -66,6 +69,16 @@ public static void MissionNameMarkerVMConstructorPostfix(MissionNameMarkerVM __i
             ____heightOffset = new Vec3(0, 0, 4, -1);
         }
 
+        [UsedImplicitly, HarmonyPatch(typeof(DefaultClanTierModel), nameof(DefaultClanTierModel.GetCompanionLimit))]
+        public static void Postfix(ref int __result)
+        {
+            if (GlobalCommonConfig.Get() != null && GlobalCommonConfig.Get().BreakCompanionLimit)
```

Comment:
Use `BLTAdoptAHeroModule.CommonConfig` instead, `.Get()` parses it from json each time.